proposals/0305-swiftpm-binary-target-improvements.md

```diff
@@ -5,7 +5,7 @@
 * Review Manager: [Tom Doron](https://github.com/tomerd)
 * Status: **Active Review (March 11 - 23 2021)**
 * Implementation: Available in [recent `main` snapshots](https://swift.org/download/#snapshots) behind the flag `SWIFTPM_ENABLE_PLUGINS`
- 
```

Comment:
could you please add a "preview version" link like https://github.com/apple/swift-evolution/pull/1309/files#diff-46ff5db64a512c3fe5d3d4097dcb216fdd752d194a775e43c88cd134f94da1d3R7

Reply:
also please add link to the pitch and discussion threads like https://github.com/apple/swift-evolution/pull/1309/files#diff-46ff5db64a512c3fe5d3d4097dcb216fdd752d194a775e43c88cd134f94da1d3L7